test/e2e/network/netpol/network_policy.go

```diff
@@ -0,0 +1,884 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package netpol
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	"k8s.io/kubernetes/test/e2e/network"
+	utilnet "k8s.io/utils/net"
+
+	"github.com/onsi/ginkgo"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+const (
+	addSCTPContainers  = false
+	isVerbose          = true
+	useFixedNamespaces = false
+
```

Comment:
this constants seems to modify the test behaviour, how are they mean to be used and how do they affect the tests behaviour?

Reply:
in general, these are not really meant to be modified by end users or test-runners, but possibly by developers... no strong opinion here can remove them if theres any concern

Reply:
Ahh ok, is an internal detail, got it ...